explainaboard/resources/dataset_custom_features.json

```diff
@@ -3,6 +3,7 @@
     "custom_features": {
       "example": {
         "label": {
+          "cls_name": "Value",
```

Comment:
We need a doc to detail how to specify `cls_name` and `dtype` in the future when we aim to document `how to add customized features through JSON config?` We potentially don't need to do this now since we haven't finalized a good way to add customized features.

Reply:
It'd be better to make an issue of this.

Reply:
Done: https://github.com/neulab/ExplainaBoard/issues/464